training/Exercise06_Gibbs.ipynb

```diff
@@ -436,11 +508,11 @@
    "source": [
     "## 2.3 Gibbs Sampler\n",
     "\n",
-    "The hierarchical model above has some important properties that we\n",
+    "The hierarchical Bayesian model above has some important properties that we\n",
     "can exploit to make the sampling more efficient. First, note that\n",
     "the Gamma distribution are conjugate priors for the precision of\n",
```

Comment:
Something wrong about the sentence "Gamma distribution are conjugate priors for the precision of the Gaussian distribution". Are should be is I guess, and is it a conjugate PRIOR or conjugate DISTRIBUTION? It is not conjugate to the PRECISION of the Gaussian distribution, is it, rather to the Gaussian distribution?

Reply:
I tried to update it!